lib/ohai/plugins/aix/network.rb

```diff
@@ -45,76 +46,78 @@ def hex_to_dec_netmask(netmask)
     network Mash.new unless network
     network[:interfaces] = Mash.new unless network[:interfaces]
 
-    # :default_interface, :default_gateway - route -n get 0
-    so = shell_out("netstat -rn |grep default")
-    so.stdout.lines.each do |line|
-      items = line.split(' ')
-      if items[0] == "default"
-        network[:default_gateway] = items[1]
-        network[:default_interface] = items[5]
+    # We unfortunately have to do things a bit different here, if ohai is running
+    # within a WPAR. For instance, the WPAR isn't aware of some of its own networking
+    # minutia such as default gateway/route.
+    unless shell_out("uname -W").stdout.to_i > 0
```

Comment:
Nah, it's a funky AIX-specific flag for `uname`:
```
-W    Displays the static workload partition identification number. If the uname command runs in the Global environment, a value of zero is displayed.
```

Stole it from Julian: https://github.com/chef/ohai/blob/master/lib/ohai/plugins/aix/virtualization.rb#L34-L36